libraries/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ add_subdirectory( libfc )
 add_subdirectory( builtins )
 add_subdirectory( softfloat )
 add_subdirectory( wasm-jit )
-add_subdirectory( chainbase )
+add_subdirectory( chaindb )
```

Comment:
Need to update `CMakeModules/EosioTesterBuild.cmake.in`

Reply:
https://github.com/AntelopeIO/spring/pull/1449/commits/3f7cfc65b654e4866522fd20ba4b86ded66b5387